src/tribler-common/tribler_common/sentry_reporter/sentry_reporter.py

```diff
@@ -165,6 +167,31 @@ def send(event, post_data, sys_info):
 
             return event
 
+    @staticmethod
+    def send_exception_with_confirmation(exception):
+        """Send exception with a confirmation dialog.
+        This method should be used in case standard GUI "sending reports mechanism"
+        is no available.
+
+        Args:
+            exception: can be either exception and any other object
+        """
+        SentryReporter._logger.debug(f"Send exception with confirmation: {exception}")
+
+        _ = QApplication(sys.argv)
+        messagebox = QMessageBox(icon=QMessageBox.Critical, text=f'{exception}.')
+        messagebox.setWindowTitle("Error")
+        messagebox.exec()
+
+        messagebox = QMessageBox(
+            icon=QMessageBox.Question,
+            text='Do you want to sent the crash report to the Tribler team? ' 'All the data will be anonymized.',
```

Comment:
Please, verify the correctness of the string.

Reply:
- "sent" -> "send"

Nits:

- "the crash report" -> "this crash report"

 - Perhaps you could make the second sentence a bit less sterile by stating something like:

"We anonymize all your data, who you are and what you downloaded."

Reply:
@qstokkink thank you!